### ISSUE TITLE
Add config property to change default severity

### ISSUE THREAD

**Issue by davidtheclark** [stylelint/stylelint#1475]
See discussion around https://github.com/stylelint/stylelint-config-standard/issues/23#issuecomment-226451861.

```js
{
  extends: "stylelint-config-standard",
  defaultSeverity: "warning"
  rules: {
    "color-no-invalid-hex": [ true, { severity: "error" } ]
  }
}
```

I think this may be pretty straightforward to implement.

cc @thomas-darling
